src/containers/AddTodo.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import { addTodo } from '../actions'
+import { addTodo } from '../features/TodoSlice'
 
 const AddTodo = ({ dispatch }) => {
   let input
```

Comment:
its better habit to add the `;` after declaring a variable but not setting a value to it